src/NLog.Extensions.Logging/LogEventInfoExtensions.cs

```diff
@@ -0,0 +1,36 @@
+using Microsoft.Extensions.Logging;
+
+namespace NLog.Extensions.Logging
+{
+
+    public static class LogEventInfoExtensions {
+
+
+        /// <summary>
+        /// Gets the <see cref="Microsoft.Extensions.Logging.EventId"/>. If it was not included in the log 
+        /// event, then uses the default value.
+        /// </summary>
+        /// <param name="logEvent"></param>
+        /// <param name="defaultValue"></param>
+        /// <returns></returns>
+        public static EventId getEventId(this LogEventInfo logEvent, EventId defaultValue = default(EventId))
+        {
+
+            if (!logEvent.HasProperties)
+                return defaultValue;
+
+            // setup key names the same way "NLogLogger.cs" does it
+            var options = new NLogProviderOptions();
```

Comment:
this is unfortunately not correct. Now you will only get the default options.

This can be removed, see next note